pymfe/landmarking.py

```diff
@@ -717,7 +718,7 @@ def ft_one_nn(
             sample_inds: t.Optional[np.ndarray] = None,
             random_state: t.Optional[int] = None,
     ) -> np.ndarray:
-        """Evaluate the performance of the 1-nearest neighbor classifier.
+        """Performance of the 1-nearest neighbor classifier.
```

Comment:
Maybe use upper case initial letters in 1-NN's name.

Reply:
Fixed in 5add14c